docs/known-issues.md

```diff
@@ -0,0 +1,26 @@
+# Known Issues
+
+## My game window is flickering white for a split second on native when I start the game
+
+The game window is created before the GPU is ready to render everything.
+This means that it will start with a white screen for a little bit.
+The workaround is to [start the Window invisible](https://github.com/bevyengine/bevy/blob/release-0.14.0/examples/window/window_settings.rs#L29-L32)
+and then [make it visible after an arbitrary amount of frames](https://github.com/bevyengine/bevy/blob/release-0.14.0/examples/window/window_settings.rs#L56-L64).
+
+## My character or camera is not moving smoothly
+
+A character not moving smoothly is often caused by the movement updates being tied to the frame rate.
+See the [physics_in_fixed_timestep](https://github.com/bevyengine/bevy/blob/main/examples/movement/physics_in_fixed_timestep.rs) example
+for how to fix this.
+
+A camera not moving smoothly is pretty much always caused by the camera position being tied too tightly to the character's position.
+To give the camera some inertia, use the [`smooth_nudge`](https://github.com/bevyengine/bevy/blob/main/examples/movement/smooth_follow.rs#L127-L142)
+to interpolate the camera position towards its target position.
+
+## My audio is stuttering on web
```

Comment:
I'd put this one first since it's the biggest issue.

Reply:
[`726758a` (#176)](https://github.com/TheBevyFlock/bevy_quickstart/pull/176/commits/726758acca0ff344150a61c1d7c71f4624073c85)